### DISCUSSION TITLE
Error update

### DISCUSSION THREAD

**Discussion by djmafiaem** [AzuraCast/AzuraCast#6154] [General]
Hello, I have a problem, when I want to update I get this error: ConnectException at /var/azuracast/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php L210: cURL error 6: Could not resolve host: updater ( see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://updater:8080/v1/update
Do you have any ideas how to solve it? I have Rolling Release #29d12b7 (2023-03-09 0:17) • Docker • PHP 8.2

**Comment by Dragondarn1993** [comment]
Did you modify the default Docker network (azuracast_default) with a docker-compose override.yml by any chance? If yes, that is the error. The updater only seems to work when Azuracast and the updater are in azuracast_default.

**Reply by djmafiaem** [reply]
to modify, I didn't modify anything at all.... I don't know what the problem was... but how I managed to do it is with:
Interactive Updates (Recommended)
cd /var/azuracast
./docker.sh update-self
./docker.sh update


and it worked.... and with the new update I no longer had the problem with that error.

**Comment by ratarasta** [comment]
Getting the same problem

**Reply by djmafiaem** [reply]
I realized why this error occurs in the browser, when I give an update from time to time to debian linux and in the meantime an update to AzuraCast also appears, then this error appears .. but to fix it so that I don't have problems, I do it manually through :
cd /var/azuracast
./docker.sh update-self
./docker.sh update

and the problem with that error was solved.

**Comment by ivellios1988** [comment]
I just got the same error message when I tried to update from 0.18.1 to 0.18.3.

`ConnectException at /var/azuracast/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php L210: cURL error 6: Could not resolve host: updater (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://updater:8080/v1/update`

**Reply by djmafiaem** [reply]
I solved it by updating Putty using this code:
cd /var/azuracast
./docker.sh update-self
./docker.sh update

**Reply by ivellios1988** [reply]
Yeah I do it each time the web updater fails to work.

**Comment by flashradio** [comment]
Same to me. I create another station and still have the same problem with that update once you restart the station

**Comment by flashradio** [comment]
The latest update has driven Azura admin features crazy

**Comment by flashradio** [comment]
Ok, thanks.
